inst/schema/DownloadSubmitRequest.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "note": {
+      "type": "object",
+      "properties": {
+        "name": { "type": "string" },
+        "updated": { "type": "string" },
+        "note": { "type": "string" }
+      },
+      "additionalProperties": false,
+      "required": [ "name", "updated", "note" ]
+    }
+  },
+  "type": "object",
+  "properties": {
+    "notes" : {
+      "type": "object",
+      "properties": {
+        "project_notes": { "$ref": "#/definitions/note" },
+        "version_notes": {
```

Comment:
Is this idea here that the notes posted should include version notes from this and all previous versions? That's a little tricky since, as we've discussed, we don't really have the notion of version parentage..

Reply:
Was thinking from this version and all other versions of this project, show it in date order. We don't have the trail of parentage but can we get this? Looks like db has `project_version` table so would we be able to get all `project_version` notes with this `project_id`?

Reply:
Yes, we could use version updated date to order the versions. Would you want to include later versions though, if this isn't the latest? In that case, should we highlight which version is this version.

Reply:
Good question - I will ask raise with Jeff/Rachel and make a note in the front end ticket